machine/v1/types_controlplanemachineset.go

```diff
@@ -284,6 +289,17 @@ type GCPFailureDomain struct {
 	Zone string `json:"zone"`
 }
 
+// OpenStackFailureDomain configures failure domain information for the OpenStack platform.
+type OpenStackFailureDomain struct {
+	// ComputeAvailabilityZone is the nova availability zone in which the OpenStack machine provider will create the VM.
+	// +optional
```

Comment:
Can you also mention what happens when the field is not specified?

What if `ComputeAvailabilityZone` is empty but `StorageAvailabilityZone` is not?

Reply:
I think I answered that above.